src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/WikiBasePageObject.java

```diff
@@ -151,12 +151,21 @@ public String getUrl() {
     return driver.getCurrentUrl();
   }
 
+  public boolean isBannerNotificationContainerPresent(){
+    return isElementPresent(BANNER_NOTIFICATION_CONTAINER);
+  }
+
   /**
-   * Checks if container containing all banner notification is present, it may be present, but may have 0px height
-   * when no notification is displayed
+   * Checks if present NOW
+   *
+   * TODO: Move this logic to better place (WebElement?)
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1135)